src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -28,8 +28,8 @@ library GPv2Encoding {
     /// settlement.
     struct Trade {
         Order order;
-        uint8 sellTokenIndex;
-        uint8 buyTokenIndex;
+        uint256 sellTokenIndex;
+        uint256 buyTokenIndex;
```

Comment:
Is this an unrelated change or is `u8` just not part of the `abiEncoder`?

Reply:
It is, its just there is no point in restricting the size of the type anymore as it just costs more gas and doesn't really give you anything.